RecoLocalMuon/RPCRecHit/src/RPCRecHitStandardAlgo.cc

```diff
@@ -24,18 +25,32 @@ bool RPCRecHitStandardAlgo::compute(const RPCRoll& roll,
   const float fstrip = (roll.centreOfStrip(cluster.firstStrip())).x();
   const float lstrip = (roll.centreOfStrip(cluster.lastStrip())).x();
   const float centreOfCluster = (fstrip + lstrip)/2;
+  const double y = cluster.hasY() ? cluster.y() : 0;
+  Point = LocalPoint(centreOfCluster, y, 0);
```

Comment:
shouldn't x be adjusted for the position here as well, in the same way as on line 46 https://github.com/cms-sw/cmssw/pull/17289/files#diff-d8cf3476ce23755760589e7a32ab4450R46 ?